.docker/prometheus/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+_term() {
+  kill -TERM "$child" 2>/dev/null
+}
+
+trap _term SIGTERM
+
+/bin/envsubst < /etc/config/prometheus_template.yml > /etc/prometheus/prometheus.yml
+/bin/prometheus \
+  --config.file=/etc/prometheus/prometheus.yml \
+  --storage.tsdb.path=/prometheus \
+  --web.console.libraries=/usr/share/prometheus/console_libraries \
+  --web.console.templates=/usr/share/prometheus/consoles &
```

Comment:
This is the defaut command from https://github.com/prometheus/prometheus/blob/e21ff116d8eb7a9bd4d889214d72e3252f3e6814/Dockerfile#L25-L28

I just added envsubst for templating and graceful shutdown `docker` -> `sh` -> `prometheus`